activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceControllerImplIT.java

```diff
@@ -177,7 +196,10 @@ void startProcess() throws Exception {
 
         mockMvc
             .perform(
-                post("/v1/process-instances").contentType(APPLICATION_JSON).content(mapper.writeValueAsString(cmd))
+                post("/v1/process-instances")
+                    .contentType(APPLICATION_JSON)
+                    .content(mapper.writeValueAsString(cmd))
+                    .with(csrf())
```

Comment:
What is the reason to add CSRF for REST API calls test? The Activiti Runtime Rest API never had the constraint to have CSRF token in cookies..